### DISCUSSION TITLE
ChatGPT replies not generating... error message.

### DISCUSSION THREAD

**Discussion by Digical1** [GorvGoyl/ChatGPT-Writer#39] [Report Bugs and Issues]
Too many requests in 1 hour. Try again later.
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

**Comment by iamshreyashpatel** [comment]
i am facing same issue since morning . unable to use anything

**Comment by NaveedKashif** [comment]
I face the same problem! Any solution?

**Comment by GorvGoyl** [comment]
looks like it's working for me now.. could you try again. If you are still facing the issue then please share a video recording so that I can debug.

**Reply by iamshreyashpatel** [reply]
It's not working for me yet ( Video Attached ), and Gaurav, I also want to say that you've created a wonderful tool. Even if you make this a paid tool in the future. It is no problem for me to pay. I am also waiting for the special version for which a Google form is available



https://user-images.githubusercontent.com/124081056/215974737-ef05a54e-88f7-4a44-a20a-4bbfb4933b79.mp4

**Comment by GorvGoyl** [comment]
fixed in version v2.1.4.. please wait for it to go live on chrome extension store https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/

**Reply by NaveedKashif** [reply]
Thank you Gorv

**Comment by Digical1** [comment]
It's working now. Thank you Gorv !!
